README.md

```diff
@@ -124,6 +124,21 @@ After running the script to install system dependencies, build the project with
 
     stack test
 
+## Build on CentOS (native OS with stack)
+
+1. install `stack` with version >1.4.0
+
+2. run the [install_centos_dependencies.sh](./tools/install_centos_dependencies.sh)
+script in the `tools/` directory to install dependencies.
```

Comment:
Let's rephrase this in a shorter way:

2. run [tools/install_centos_dependencies.sh](./tools/install_centos_dependencies.sh)
to install dependencies.